tests/suites/0_stateless/02_0014_function_maths.result

```diff
@@ -0,0 +1,4 @@
+ERROR 1105 (HY000) at line 4: Code: 7, displayText = Expected numeric types, but got String.
```

Comment:
-- {ErrorCode 1105}

To match the error code.

Reply:
fixed, thank you